grammar.js

```diff
@@ -176,6 +178,9 @@ module.exports = grammar({
 
         $.variable_open,
         $.variable_close,
+
+        $.inline_link_target_open,
+        $.inline_link_target_close,
```

Comment:
This is slightly inconsistent right now:
- attached markup is generally treated like this
- BUT: link descriptions and texts are not treated the same way...
- this is somewhat inbetween. How shall we unify this?